snp-sites/snp-sites.cwl

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+$namespaces:
+  edam: http://edamontology.org/
+  s: http://schema.org/
+
+requirements:
+  InlineJavascriptRequirement: {}
+
+inputs:
+  actg_only:
+    doc: only output columns containing exclusively ACTG
+    type: boolean?
+    inputBinding:
+      prefix: -c
+  monomorphic_sites:
+    doc: output monomorphic sites (suitable for BEAST)
+    type: boolean?
+    inputBinding:
+      prefix: -b
+  output_options:
+    type:
+    - type: record
+      name: output_fasta
+      fields:
+        output_fasta:
```

Comment:
Is the tool capable of producing both output types at the same time? 

If that isn't too much overhead, then I suggest doing so always